_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -141,55 +141,58 @@ $shadow: transparentize(palette(grayscale, black-t), .5);
 // ----------------------------
 // #TYPOGRAPHY
 // ----------------------------
+
+$ux-pattern-library-font-path: '../fonts' !default;
```

Comment:
@clintonb I *disagree* with the prefix. The Pattern Library HTML/Sass code is not intended to be used with Bootstrap nor any other UI Frameworks that handle the same level of rendering as it does.

What you folks are doing with Oscar and Otto is not the intended use of the code in this repo and I don't think this is the time to inject exceptions or special cases based on that work. If you need to handle conflicts within your project, I'd recommend that you do that within your project's codebase and Sass compilation.